bundles/org.openhab.binding.energidataservice/README.md

```diff
@@ -579,6 +579,69 @@ var result = edsActions.calculateCheapestPeriod(time.Instant.now(), time.Instant
 
 :::
 
+::: tab JRuby
+
+```ruby
+energi = things["energidataservice:service:energidataservice"]
```

Comment:
I don't know JRuby, but it seems like you can directly access actions from a Thing reference, so I understand that the "Actions" suffix used for other languages doesn't make sense here - correct? But perhaps we could still reuse the abbreviation `eds` (for Energi Data Service) rather than `energi` (English: energy)?

Reply:
> it seems like you can directly access actions from a Thing reference, so I understand that the "Actions" suffix used for other languages doesn't make sense here - correct?

Correct.

> perhaps we could still reuse the abbreviation eds (for Energi Data Service) rather than energi (English: energy)?

I'll change it to `eds`